requirements.txt

```diff
@@ -5,3 +5,4 @@ wagtailsurveys==0.1.1
 wagtail-personalisation==0.9.1
 psycopg2
 html5lib==0.9999999
+Unidecode==0.04.16
```

Comment:
Why do we need this pin?

Reply:
@miltontony one of our dependencies require this version of unidecode, so if we don't add this we get a version conflict as we try to install 0.04.21